### PR TITLE
Changes in language and code visual

### PR DESCRIPTION
**Pull request by hohky**
This pull request addresses improvements in the code base by refining the language used across comments and documentation to enhance clarity. Additionally, I have made visual adjustments to the code formatting for better readability and consistency. These changes ensure that both developers and users can navigate the project more easily while maintaining code standards.

- Updated variable names for better understanding.
- Improved comment descriptions for clarity.
- Reformatted sections of code to align with the coding style guide.
- Please review the changes and let me know if further adjustments are needed.

http://www.askpython.com/python-modules/what-does-dot-mean-in-python-import

### CODE REVIEW

File: .yarnrc
Comment (zamitto) [inline .yarnrc:1]: Why does this file exist?

File: .yarn/releases/yarn-1.22.22.cjs
Comment (zamitto) [inline .yarn/releases/yarn-1.22.22.cjs:1]: Why does this file exist?

Comment (hohky) [reply .yarn/releases/yarn-1.22.22.cjs:1]: I didn't mean to, I'm working on it right now, I'm sorry!!!

**Comment by hohky** [comment]
I've already deleted the unnecessary files.

**Review by zamitto — APPROVED** [review]

**Pull request merged**